models/datastore_test.go

```diff
@@ -0,0 +1,113 @@
+package models_test
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/atmiguel/cerealnotes/models"
+	"github.com/atmiguel/cerealnotes/test_util"
+)
+
+var postgresUrl = "postgresql://localhost/cerealnotes_test?sslmode=disable"
+
+const noteTable = "note"
+const publicationTable = "publication"
+const noteToPublicationTable = "note_to_publication_relationship"
+const noteToCategoryTable = "note_to_category_relationship"
+const userTable = "app_user"
+
+var tables = []string{
+	noteToPublicationTable,
+	publicationTable,
+	noteToCategoryTable,
+	noteTable,
+	userTable,
+}
+
+func ClearAllValuesInTable(db *models.DB) {
+	for _, val := range tables {
+		if err := ClearValuesInTable(db, val); err != nil {
+			panic(err)
+		}
+	}
+}
+
+func ClearValuesInTable(db *models.DB, table string) error {
```

Comment:
It would require a lot more work. Right now i'm already struggling with having 1 test database. I would have to create go code that would spin up new database + tables on the fly. And that would mean moving my source of truth from migrations to go code, which i'm not really interested in doing right now.

Reply:
if it's too much effort then don't worry about it. It sounds like the effort will outweigh the reward. But if we ever use gorm, it will make this easy for us https://godoc.org/github.com/jinzhu/gorm#DB.CreateTable

Any reasonable orm should make this easy for us. gorm is just the only one i've read about in go